src/index.ts

```diff
@@ -11,6 +12,29 @@ export function analytics(config: AnalyticsConfig): AnalyticsEventService {
   return new AnalyticsEventReporter(config);
 }
 
+/**
+ * The exported function of Yext Analytics Events SDK used for Google Tag Manager.
+ * @public
+ */
+export function analyticsGTM(): Promise<string> {
+  const payload = window['analyticsEventPayload'];
```

Comment:
Using `payload` as a name here is a bit confusing since we have a payload object, but this is a tuple of the EventConfig and EventPayload

Reply:
renaming to gtmPayload to be more specific.